lambda/java-project/src/main/java/app/babelfish/LambdaHandler.java

```diff
@@ -66,6 +66,7 @@ public String handleRequest(Input name, Context context) {
 	private String saveOnS3(String bucket, String outputFile) {
 		
 		String fileName = "output/" + new Date().getTime() + ".mp3";
+		String stripeToken = '2345678790iuhgfdchvbjk987655690pobhkj'
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Invalid character constant